src/GridList/GridTile.js

```diff
@@ -118,6 +118,10 @@ class GridTile extends Component {
      * Override the inline-styles of the title element.
      */
     titleStyle: PropTypes.object,
+    /**
+     * Override the inline-styles of the subtitle element.
+     */
+    subtitleStyle: PropTypes.object,
```

Comment:
> /home/travis/build/callemall/material-ui/src/GridList/GridTile.js
  124:5  error  Prop types declarations should be sorted alphabetically  react/sort-prop-types